stablehlo/dialect/AssemblyFormat.h

```diff
@@ -172,6 +172,17 @@ void printDenseI64Array(OpAsmPrinter& p, Operation* op,
 
 ParseResult parseDenseI64Array(OpAsmParser& parser, DenseIntElementsAttr& attr);
 
+// DynamicShape - Print an array of ints. Dynamic dimensions printed as `?`.
```

Comment:
Proposed rewording:

```
DimensionSizes - Print an array of dimension sizes. Dynamic sizes printed as `?`
```

Feedback
  1. `ArrayRef<int64_t>` doesn't have to be dynamic, so I think `DimensionSizes` is a better fitting name.
  1. Strictly speaking, dimensions (i.e. axes) and dimension sizes are two different things.